src/assets/js/protocol/protocol-menu.js

```diff
@@ -0,0 +1,311 @@
+
+            link = items[i].querySelector('.mzp-c-menu-title');
+            link.addEventListener('click', Menu.onClickWide, false);
+
+            close = items[i].querySelector('.mzp-c-menu-button-close');
+            close.addEventListener('click', Menu.onCloseButtonClick, false);
+        }
+    };
+
+    /**
+     * Unbind events for wide viewports.
+     */
+    Menu.unbindEventsWide = function() {
+        var items = document.querySelectorAll('.mzp-c-menu-category.mzp-js-expandable');
+        var link;
+        var close;
+
+        for (var i = 0; i < items.length; i++) {
+            items[i].removeEventListener('mouseenter', Menu.onMouseEnter, false);
+            items[i].removeEventListener('mouseleave', Menu.onMouseLeave, false);
+            items[i].removeEventListener('focusout', Menu.onFocusOut, false);
+
+            link = items[i].querySelector('.mzp-c-menu-title');
+            link.removeEventListener('click', Menu.onClickWide, false);
+
+            close = items[i].querySelector('.mzp-c-menu-button-close');
+            close.removeEventListener('click', Menu.onCloseButtonClick, false);
+        }
+    };
+
+    /**
+     * Bind events for small viewports.
+     */
+    Menu.bindEventsSmall = function() {
+        var items = document.querySelectorAll('.mzp-c-menu-category.mzp-js-expandable .mzp-c-menu-title');
+
+        for (var i = 0; i < items.length; i++) {
+            items[i].addEventListener('click', Menu.onClickSmall, false);
+        }
+    };
+
+    /**
+     * Unbind events for small viewports.
+     */
+    Menu.unbindEventsSmall = function() {
+        var items = document.querySelectorAll('.mzp-c-menu-category.mzp-js-expandable .mzp-c-menu-title');
+
+        for (var i = 0; i < items.length; i++) {
+            items[i].removeEventListener('click', Menu.onClickSmall, false);
+        }
+    };
+
+    /**
+     * Set initial ARIA menu panel states.
+     */
+    Menu.setAria = function() {
+        var items = document.querySelectorAll('.mzp-c-menu-category.mzp-js-expandable .mzp-c-menu-title');
+
+        for (var i = 0; i < items.length; i++) {
+            items[i].setAttribute('aria-expanded', false);
+        }
+    };
+
+    /**
+     * Enables simplified menu using pure CSS hover states.
+     */
+    Menu.cssFallback = function() {
+        var menu = document.querySelector('.mzp-c-menu');
+        var currentClassName = menu.className;
+        menu.className = currentClassName.replace(/mzp-c-menu/, 'mzp-c-menu mzp-c-menu-basic');
+    };
+
+    /**
+     * Basic feature detect for 1st class menu JS support.
+     */
+    Menu.isSupported = function() {
+        return typeof window.matchMedia !== 'undefined' &&
+               window.matchMedia('all').addListener &&
+               'classList' in document.createElement('div');
+    };
+
+    /**
+     * Initialise menu.
+     * @param {Object} options - configurable options.
+     */
+    Menu.init = function(options) {
```

Comment:
I don't see where this is called, except in the tests. Is this function just here for the tests?